packages/frontend/src/__tests__/hooks/useCreateComment.test.tsx

```diff
@@ -0,0 +1,96 @@
+import { renderHook, act } from '@testing-library/react-hooks'
```

Comment:
have to replace '@testing-library/react-hooks' with '@testing-library/react' for supporting React 18.

https://github.com/testing-library/react-hooks-testing-library#a-note-about-react-18-support